transport/nio/src/main/java/org/kaazing/gateway/transport/nio/internal/AbstractNioAcceptor.java

```diff
@@ -644,7 +652,6 @@ private IoSession getTcpBridgeSession(IoSession session) {
                     return (IoSession) session.getAttribute(TCP_SESSION_KEY);
                 }
 
-
                 @Override
                 protected void doExceptionCaught(IoSessionEx session, Throwable cause) throws Exception {
                     //TODO: consider tcpBridgeSession.reset as an alternate impl.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaazing.wan:9000/coding_rules#rule_key=squid%3AS1135)